### ISSUE TITLE
Savescumming performance drop

### ISSUE THREAD

**Issue by sanscriptum** [Mtarnuhal/FrozenCookies#74]
After savescumming for quite a bit (that is reloading the save over and over again), I notice huge performance deterioration. FrozenCookies is the only script I run, and I have "no buildings" blacklist for the autobuy. I also have every upgrade purchased. Perhaps there's a memory leak somewhere, it usually fixes itself after reloading the tab. This is the only Cookie Clicker userscript that causes this so far I've noticed.

**Comment by Mtarnuhal** [comment]
By any chance do you have logging or tracking turned on in the options?

**Comment by sanscriptum** [comment]
No, I made sure to turn it off. The only resource intensive thing I have (I think) is autobuy. I included my save.
[LovelyFactoryBakery.txt](https://github.com/Mtarnuhal/FrozenCookies/files/5861628/LovelyFactoryBakery.txt)
